config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -3244,7 +3199,57 @@ presubmits:
           value: /workspace/k8s.io/kubernetes
         - name: METADATA_BANDAID
           value: "true"
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-master
```

Comment:
Would it make sense to split the new job additions into another commit/PR so that they can be tracked?
Maybe we can add Comment/description?

Reply:
this file/config is completely generated from other files, we want to avoid handcraft it.

Reply:
may be we should add description field into the job configs. This will help generate description too.

Reply:
there's
```
# Autogenerated by genjobs.go, do NOT edit!
# see genjobs.go, which you can run with hack/update-config.sh
```
at the top... maybe we don't even need to check-in this config? /shrug

feel free to open an issue/PR if you think it bothers you.